central/graphql/handler/handler_benchmark_test.go

```diff
@@ -162,7 +162,7 @@ func getGraphQLServer(testHelper *testutils.ExportServicePostgresTestHelper) (*h
 	}
 	loaders.RegisterTypeFactory(reflect.TypeOf(storage.Deployment{}), deploymentFactory)
 	imageFactory := func() interface{} {
-		return loaders.NewImageLoader(resolver.ImageDataStore)
+		return loaders.NewImageLoader(resolver.ImageDataStore, testHelper.ImageView)
```

Comment:
Do you think it would make sense to create a tech debt ticket for adding benchmark test to some of these VM queries?

Reply:
Yeah, I created a ticket